app/views/groups/index.html.erb

```diff
@@ -0,0 +1,46 @@
+<div class="container mt-4">
+  <div class="row mb-4 text-end justify-content-end bg-primary p-2">
+    <%= link_to 'Most Recent', groups_path(sort: 'recent'), class: 'text-decoration-none text-light mw-100' %>
+    <%= link_to 'Most Ancient', groups_path(sort: 'ancient'), class: 'text-decoration-none text-light mw-100' %>
+  </div>
+
+  <div class="row gap-2 justify-content-center">
+    <% @groups.each do |group| %>
+      <div class="col-md-4 col-lg-3 card p-0">
+        <%= link_to group_entities_path(group), class: 'text-decoration-none text-secondary' do %>
+          <div class="d-flex">
+            <div class="col-3">
+              <img src=<%= group.icon %> class='img-tumbnail'>
+            </div>
+            <div class="col-6">
+              <div class="col-12">
+                <%= group.name %>
+              </div>
+              <div class="col-12 mt-4">
+                <%= group.created_at.strftime('%b %d %Y') %>
+              </div>
+            </div>
+            <div class="col-3">
+              <% sum = @group_entity_sums[group.id] || 0 %>
+              <p class='text-bold text-dark text-end pe-1'><%= number_to_currency(sum, unit: "$") %></p>
+            </div>
+          </div>
+          <div class="card-footer">
+            <%= form_with(model: group, url: group_path(group), method: :delete, data: { confirm: 'Are you sure?' }, class: 'd-flex gap-2 justify-content-end') do |form| %>
+              <%= link_to edit_group_path(group), class: 'btn btn-sm btn-primary mr-2' do %>
+                <i class="fas fa-edit"></i>
+              <% end %>
+              <%= form.button type: 'submit', class: 'btn btn-danger btn-sm' do %>
+                <i class="fa fa-trash"></i>
+              <% end %>
+            <% end %>
+          </div>
+        <% end %>
+      </div>
+    <% end %>
+  </div>
+  
+  <div class="container bottom-element">
+    <%= link_to "Add Group", new_group_path(), class: "btn btn-primary w-100" %>
```

Comment:
- [ ] While you did well placing this button to the bottom of the page, kindly note that it is a requirement to make sure that we stick to the provided design typography which includes both the color and layout. KIndly ensure just as mentioned by both reviewers that the button's color is changed to green and not blue as it currently is.